File: day02/Nilesh_18311/Car.java
package myOwnAutoShopProject;

public class Car {
	int speed;
	double regularPrice;
	String color;
	double sale;
	double cost;
	
	Car(double reg,String c,double sal)
	{
		this.regularPrice=reg;
		this.color=c;
		this.sale=sal;
	}
	Car()
	{
		
	}
	
	double getSalePrice()
	{
		 cost=regularPrice- regularPrice*sale;
		 double salePrice=regularPrice*sale;
		 System.out.println("Sale Price is:"+ salePrice);
		return cost;
	}
	public void display()
	
	{
		System.out.println("Car price with discount:"+getSalePrice());
		System.out.println("Color:"+ color);
		System.out.println();
	}
}

File: JS/Day1Assign/Car.ts
export class Car
{
    speed:number;
    regularPrice: number;
    color: string;
    
    constructor(speed:number,price:number,color:string)
    {
        this.speed=speed;
        this.regularPrice=price;
        this.color=color;
    }

    getSalePrice():number{
        
       const salePrice =this.regularPrice-this.regularPrice*.10;

        return salePrice;
    }
    display()
    {
        console.log("Speed: " +this.speed+", Regular Price: "+ this.regularPrice+ ", Color: "+ this.color);
        console.log("Sale Price: " +this.getSalePrice() );
    }
}